src/modules/email/infrastructures/email-infrastructure.module.ts

```
@@ -0,0 +1,36 @@
+          port: Number(process.env.MAIL_PORT),
+          secure: process.env.MAIL_SECURE === 'true',
+          auth: {
+            user: process.env.MAIL_USER,
+            pass: process.env.MAIL_PASS,
+          },
+        },
+        defaults: {
+          from: process.env.MAIL_FROM,
+        },
+        template: {
+          dir: join(process.cwd(), 'dist/modules/email/infrastructures/templates'),
```

ESLint

[error] Replace `process.cwd(),·'dist/modules/email/infrastructures/templates'` with `⏎············process.cwd(),⏎············'dist/modules/email/infrastructures/templates',⏎··········`

(prettier/prettier)